cli/azd/pkg/input/console.go

```diff
@@ -711,6 +700,33 @@ func NewConsole(noPrompt bool, isTerminal bool, w io.Writer, handles ConsoleHand
 	return c
 }
 
+// IsTerminal returns true if the given file descriptors are attached to a terminal,
+// taking into account of environment variables that force TTY behavior.
+func IsTerminal(stdoutFd uintptr, stdinFd uintptr) bool {
+	// User override to force non-TTY behavior
+	if os.Getenv("AZD_DEBUG_FORCE_NO_TTY") == "1" {
```

Comment:
FWIW - if we want to use `ParseBool` here, totally happy with that - the reason it was `== 1` was this was one of the first debug environment flags we added, and I had forgotten about the `ParseBool` pattern.  If anyone was wondering why we were slightly inconsistent here.

Reply:
Sure!